terraso_backend/apps/story_map/models/story_maps.py

```diff
@@ -0,0 +1,57 @@
+﻿# Copyright © 2021-2023 Technology Matters
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published
+# by the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see https://www.gnu.org/licenses/.
+
+from django.db import models
+from safedelete.models import SOFT_DELETE
+
+from apps.core.models import BaseModel, SlugModel, User
+from apps.core.models.commons import validate_name
+from apps.story_map import permission_rules as perm_rules
+
+
+class StoryMap(SlugModel):
+    # file will not be deleted in cascade
+    _safedelete_policy = SOFT_DELETE
+
+    title = models.CharField(max_length=128, validators=[validate_name])
```

Comment:
question: should we have longer titles?

Reply:
@paulschreiber same as the other answer, what is your suggestion for the correct size?